### PR TITLE
Refactor titles

### PR DESCRIPTION
**Pull request by tiwanari**

### CODE REVIEW

File: app/src/main/java/com/pileproject/drive/title/TitleActivityBase.java [hunk 1]
```diff
@@ -16,31 +16,57 @@
...
+
+    /**
+     * A method to get an intent to programming page.
+     */
+    protected abstract Intent getIntentToProgrammingPage();
+
+    /**
+     * A method to get an intent to preference page.
+     */
+    protected abstract Intent getIntentToSetttingPage();
```
Comment (mandaiy) [inline app/src/main/java/com/pileproject/drive/title/TitleActivityBase.java]: typo

File: app/src/nxt/java/com/pileproject/drive/title/NxtTitleActivity.java [hunk 2]
```diff
@@ -18,35 +18,21 @@
...
-            }
-        });
+    protected Intent getIntentToProgrammingPage() {
+        return new Intent(getApplicationContext(), NxtProgrammingActivity.class);
+    }
 
-        setupToolbar();
+    @Override
+    protected Intent getIntentToSetttingPage() {
+        return new Intent(getApplicationContext(), NxtSettingActivity.class);
```
Comment (mandaiy) [inline app/src/nxt/java/com/pileproject/drive/title/NxtTitleActivity.java]: Now the `NxtSettingActivity` has nothing. We can change it to `SettingActivity`

(The content of setting page can be specified with {nxt,ev3,pile}/res/xml/preferences.xml)

Comment (tiwanari) [inline app/src/nxt/java/com/pileproject/drive/title/NxtTitleActivity.java]: Wow, I hadn't noticed that. Thanks.

File: app/src/main/java/com/pileproject/drive/title/TitleActivityBase.java (hunk 1 above)
Comment (tiwanari) [inline app/src/main/java/com/pileproject/drive/title/TitleActivityBase.java]: Sorry & Thanks.

**Comment by tiwanari** [comment]
Fixed. Please check again @myusak .

**Comment by mandaiy** [comment]
ship it!

**Pull request merged**